tests/parsers/test_vasp.py

```diff
@@ -62,7 +62,20 @@ def test_perov(self):
    
         total_mag = parser.get_total_magnetization()
         assert(total_mag.scalars[0].value == 3.9999992)
-        assert(total_mag.units == "Bohr") 
+        assert(total_mag.units == "Bohr")
+
```

Comment:
Why not use assertEqual?

Reply:
`assertEqual` is part of unittest and some people (such as myself) prefer different test harnesses (e.g. py.test).  Those people often just use `assert`.  In this case, the value of `assertEqual` is mostly lost anyways, since string equality is exact and self-describing.

Reply:
Fair enough. I don't think it's worth holding up the show for this change. I've adjusted my review to "approve," in cast that's relevant. 

Out of curiosity, why do you prefer `py.test`?

Reply:
I find `py.test` to be more lightweight (you don't have to define classes, for example).  Most of my python projects are small, so I value that simplicity.

[3rd party opinions](https://www.slant.co/versus/9148/9149/~unittest_vs_pytest)